src/apps/chifra/internal/traces/handle_show.go

```diff
@@ -38,7 +38,7 @@ func (opts *TracesOptions) HandleShowTraces() error {
 			ts := rpc.GetBlockTimestamp(opts.Globals.Chain, uint64(txIds[0].BlockNumber))
 			for _, id := range txIds {
 				// Decide on the concrete type of block.Transactions and set values
-				traces, err := rpcClient.GetTracesByTransactionId(opts.Globals.Chain, uint64(id.BlockNumber), uint64(id.TransactionIndex))
+				traces, err := rpcClient.GetTracesByTransactionId(opts.Globals.Chain, uint64(id.BlockNumber), uint64(id.TransactionIndex), nil)
```

Comment:
This makes examples of calls with a cache very hard to find. What I do is this:

```
var store *cacheNew.Store = nil
...., store)
```

In this way, I can search the entire code base with " store)" to find all calls that use a cache.